### ISSUE TITLE
'dropping event for an already terminated node' error

### ISSUE THREAD

**Issue by james-callahan** [aws/aws-node-termination-handler#413]
e.g. 
```
2021/04/19 01:33:17 ERR dropping event for an already terminated node error="node: 'i-04c43476720765852' in state 'terminated': node metadata unavailable" 
```

I don't think this is an actionable or useful error.
Should it be downgraded to `info` level?

---

Environment notes:
  - EKS 1.19
  - aws-node-termination-handler 1.13
  - deployed in queue mode

**Comment by haugenj** [comment]
I think after the changes suggested from #408 this should be downgraded to a warning. Our loose standards for log levels are that handled errors should be warning level and unhandled errors should be error level

**Comment by bwagner5** [comment]
released in v1.13.1
